core/tests/test_utils.py

```diff
@@ -809,9 +937,22 @@ def __get__(self, instance, owner):
         return self
 
     def pre_call_hook(self, args):
+        """Override this to do tasks that should be executed before the
+        actual function call.
+
+        Args:
+            args: Set of arguments that the function accepts.
```

Comment:
Is args a list? If so, indicate its type as list(*).

Reply:
Done!